usr/src/test/smartos-test/README

```diff
@@ -101,3 +111,64 @@ It is important when adding or removing tests that the IPS manifests are
 updated. If new test packages are created, they should be added to
 $SRC/Makefile.testarchive.
 
+
+How We Use This:
+
+Every release we pull the release ISO and the matching test .tgz file down.
+For example:
+
+kebe(~/ws/ij-cr)[141]% mls -lt /Joyent_Dev/public/SmartOS/20230323T000605Z | egrep "iso|tests-"
+-rwxr-xr-x 1 Joyent_Dev     606420992 Mar 22 22:43 smartos-20230323T000605Z.iso
+-rwxr-xr-x 1 Joyent_Dev      12946513 Mar 22 22:43 tests-release-20230323-20230323T000605Z.tgz
+kebe(~/ws/ij-cr)[0]% 
+
+We then use a test machine (currently a VMware VM, which does cause some
+inconsistencies in the BHYVE/VMM results) which has:
+
+	16GB of RAM (though 8 would probably suffice)
+
+	A 100GB disk for `zones` (less can cause ENOSPC problems sometimes)
+
+	Three 30GB disks for setting up in DISKS for ZFS testing.  (less
```

Comment:
I think you probably mean "fewer" and not "less". "fewer" explicitly refers to quantity, while "less" may be interpreted as volume. I.e., someone may think "I'll make two 50GB disks, which is greater than the stated 90GB".

Reply:
Actually "less capacity" for the 100GB one, and "fewer" for the Three 30GB ones.